e2e/windows/e2e-helper.sh

```diff
@@ -81,10 +81,26 @@ upload_linux_file_to_storage_account() {
         log "Upload linux file successfully"
     else
         err "Failed to upload linux file. Error code is $retval."
+        exit 1
     fi
 }
 
 download_linux_file_from_storage_account() {
+    if [[ "$(check_linux_file_exists_in_storage_account)" == *"Linux file already exists in storage account."* ]]; then
+        linuxFileURL="https://${AZURE_E2E_STORAGE_ACCOUNT_NAME}.blob.core.windows.net/${WINDOWS_E2E_STORAGE_CONTAINER}/${MC_VMSS_NAME}-linux-file.zip"
+
+        export AZCOPY_AUTO_LOGIN_TYPE="MSI"
+        export AZCOPY_MSI_RESOURCE_STRING="${AZURE_MSI_RESOURCE_STRING}"
+
+        array=(azcopy_*)
+        ${array[0]}/azcopy copy $linuxFileURL file.zip
```

Comment:
can we use `az storage blob download --auth-mode login`?

Reply:
Sure. I will update it.